collective/honeypot/fixes.zcml

```diff
@@ -11,34 +11,27 @@
              collective.honeypot[fixes]
          zcml =
              collective.honeypot-fixes
-
-       In Plone 3 the above would give an error when running
-       bin/buildout, so we always load this file.  Instead, if you
-       want to disable the fixes on Plone 3, you should make sure
-       z3c.jbot is not available.  This should do the trick, unless
-       z3c.jbot is already added in some other way:
-
-         [instance]
-         eggs =
-             collective.honeypot
-
   -->
 
   <include package="z3c.jbot" />
-
-  <include zcml:condition="installed plone.app.discussion"
-           package=".discussion" />
-
-  <include zcml:condition="installed quintagroup.plonecomments"
-           package=".quinta" />
-
-  <!-- We could add zcml:condition="not-installed
-       plone.app.discussion" to oldcomments, but we always need to
-       protect these old scripts because on newer Plones they are
-       still available. -->
-  <include package=".oldcomments" />
-
-  <include zcml:condition="not-have plone-4" package=".plone3" />
-  <include zcml:condition="have plone-4" package=".plone4" />
+  <include package=".plone5" />
+
+  <adapter
+      zcml:condition="installed plone.app.discussion"
+      name="honeypot"
+      factory=".z3cform.honeypotextender.HoneypotExtender"
+      for="zope.interface.Interface
+           zope.publisher.interfaces.browser.IDefaultBrowserLayer
+           plone.app.discussion.browser.comments.CommentForm"
+      provides="plone.z3cform.fieldsets.interfaces.IFormExtender" />
+
+  <adapter
+      zcml:condition="installed collective.easyform"
```

Comment:
I included an easyform configuration here. I'd prefer that to be in easyform itself, but for that this package has to be merged and released first.